src/ucp/dt/dt_contig.c

```diff
@@ -14,10 +14,30 @@
 #include <string.h>
 
 
-size_t ucp_memcpy_pack(void *dest, void *arg)
+size_t ucp_memcpy_pack_cb(void *dest, void *arg)
```

Comment:
Renamed, in order to allow consistency of naming `ucp_mem{cpy,_type}_{un,}pack` funcs.